lib/src/scss-shared/_nx-text-helpers.scss

```
@@ -31,6 +31,25 @@
   white-space: nowrap;
 }
 
+// for specificity reasons this should be called using a custom class on the <body> tag
+// e.g. `<body className="nx-body nx-body--custom-font-size">`
```

A more realistic suggestion I think - we don't want to imply that they need to make up a separate classname strictly pertaining to the font-size adjustment.
```suggestion
// e.g. `<body className="nx-body nx-body--my-app">`
```